### DISCUSSION TITLE
いいねした記事を検索したい

### DISCUSSION THREAD

**Discussion by kobi32768** [increments/qiita-discussions#366] [要望・アイデア・不具合]
いいねした記事が多いと特定の記事を探すのが大変なのでいいねした記事の中から検索できるとうれしい
ストックした記事からは検索できるようですがいいねの方が気軽に押しがちなのでそこから検索したい

**Comment by tanaka-asami415** [comment]
@kobi32768 
ご意見ありがとうございます🙏
ご自身の記事を検索されたい等のご意見もいただいており
記事の検索性は上げたいと思っておりますので、今回いただいたご意見も合わせて検討します！💪
